module/components/Avatar/index.js

```diff
@@ -48,13 +49,17 @@ class Avatar extends React.Component {
             color,
             icon,
             src,
-            size
+            size,
+            labelText,
+            svg
         } = this.props;
 
         let defaultClasses = 'avatar avatar-rounded';
 
         return (
-            <div className={ classNames(classes, defaultClasses) } >{ icon }</div>
+            <div className={classNames(classes, defaultClasses)}>
+                {src ? (<img src={src} />) : (<span>{labelText}</span>) }
+            </div>
         );
     }
 }
```

Comment:
Please add a newline at the end for all files as per [eslint](https://eslint.org/docs/rules/eol-last) and [Airbnb](https://github.com/airbnb/javascript#whitespace--newline-at-end) spec